templates/csharp/sso-tab-ssr/Program.cs.tpl

```diff
@@ -0,0 +1,44 @@
+using {{SafeProjectName}};
+using {{SafeProjectName}}.Components;
+using {{SafeProjectName}}.Interop.TeamsSDK;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorComponents()
+    .AddInteractiveServerComponents();
+
+var config = builder.Configuration.Get<ConfigOptions>();
+builder.Services.AddTeamsFx(config.TeamsFx.Authentication);
+builder.Services.AddScoped<MicrosoftTeams>();
+
+builder.Services.AddControllers();
```

Comment:
This is because the old templates had controllers and I will do double-confirm.

Reply:
It is not possible to remove controllers as it will be used to process additional requests and the following error will occur if removed.
![error](https://github.com/OfficeDev/TeamsFx/assets/139844715/6c4421c2-243a-429d-85c4-de5d5b03d813)